src/status_im/utils/platform.cljs

```diff
@@ -28,8 +28,7 @@
 (def platform-specific
   (cond
     android? android/platform-specific
-    ios? ios/platform-specific
-    :else desktop/platform-specific))
+    ios? ios/platform-specific))
```

Comment:
I don't understand what this change does...

Reply:
ouch changed wrong line!

Reply:
done, mistakenly commited that change